app/models/publisher.rb

```diff
@@ -323,8 +332,37 @@ def paypal_locale?(locale)
     locale == 'ja'
   end
 
+  def brave_payable?
+    paypal_connection&.verified_account || uphold_connection&.payable?
+  end
+
+  def country
+    provider_country = uphold_connection&.country || paypal_connection&.country
+
+    provider_country.to_s.upcase
+  end
+
+  def valid_promo_country?
+    PromoRegistration::RESTRICTED_COUNTRIES.exclude?(country)
+  end
+
+  def promo_registrable?
```

Comment:
May we name this `may_register_promo?`

Reply:
👍 https://github.com/brave-intl/publishers/pull/2733/commits/99a33d783c7fcaa136639550816a0fcc9bd39936